linkml_runtime/linkml_model/linkml_files.py

```diff
@@ -152,7 +152,7 @@ def LOCAL_PATH_FOR(source: Source, fmt: Format) -> str:
 
 
 def GITHUB_IO_PATH_FOR(source: Source, fmt: Format, version="latest") -> str:
-    path = '/'.join([version, 'linkml_model', *_build_path(source, fmt)])
```

Comment:
This is for a URL, not an OS path, which is why it joins on `/` rather than using pathlib. Looks like the pathlib impl should be equivalent but just explaining the manual string munging